R/processing.R

```diff
@@ -5,20 +5,20 @@
 #'  datasets of the same underlying structure (i.e. same columns).
 #' @param data A data frame
 #' @return A tibble with relevant NA columns added
-#' @importFrom tibble tibble
+#' @importFrom tibble tibble add_column
+#' @importFrom rlang !!!
 add_extra_na_cols <- function(data) {
   expected_col_names <- c(
-    "cases_new", "cases_total", "deaths_new", "deaths_total",
-    "recovered_new", "recovered_total", "tested_new", "tested_total",
-    "hosp_new", "hosp_total"
+    "cases_new", "cases_total", "deaths_new", "deaths_total", "recovered_new",
+    "recovered_total", "tested_new", "tested_total", "hosp_new", "hosp_total"
+  )
+
+  new_cols <- rep(list(NA_real_), length(expected_col_names))
```

Comment:
this is slick. At some point (not now) it would be cool to make these processing steps optional.

Reply:
I don't know if we have it written down but I guess we should really tell people in the documentation for `process `what will happen to the data.....